pkl-parser/src/main/java/org/pkl/parser/GenericParser.java

```diff
@@ -719,6 +723,15 @@ private Node parseExpr(@Nullable String expectation, int minPrecedence) {
     };
   }
 
+  private Node parseUnqualifiedAccessExpr() {
+    var children = new ArrayList<Node>();
+    children.add(parseIdentifier());
+    if (lookahead == Token.LPAREN && noSemicolonInbetween() && _lookahead.newLinesBetween == 0) {
+      children.add(parseArgumentList());
+    }
```

Comment:
```suggestion
    children.add(parseIdentifier());
    ff(children);
    if (lookahead == Token.LPAREN && noSemicolonInbetween() && _lookahead.newLinesBetween == 0) {
      children.add(parseArgumentList());
    }
```
Just found another bug looking at this code: we are not fast forwarding after parsing the identifier so code like `foo.bar/*asd*/()` is not parsing, even though it works fine for the normal parser.